float32 speed
---
int64 status
